release/release_gold.sh

```diff
@@ -214,12 +217,13 @@ function print_help {
 	echo "    LIFERAY_RELEASE_NEXUS_REPOSITORY_USER (optional): Nexus user with the right to upload BOM files"
 	echo "    LIFERAY_RELEASE_PATCHER_PORTAL_EMAIL_ADDRESS: Email address to the release team's Liferay Patcher user"
 	echo "    LIFERAY_RELEASE_PATCHER_PORTAL_PASSWORD: Password to the release team's Liferay Patcher user"
+	echo "    LIFERAY_RELEASE_PREPARE_NEXT_RELEASE_BRANCH: Set to \"yes\" to proceed with branch preparation. Set to \"no\" to skip this step."
```

Comment:
@lucasmiranda0, can you change the help text a little bit to: `Set to \"yes\" to proceed with branch preparation to next release....`